src/common/sendpacket.c

```diff
@@ -211,7 +226,15 @@ static struct tcpr_ether_addr *sendpacket_get_hwaddr_pcap(sendpacket_t *) _U_;
 #undef INJECT_METHOD
 #define INJECT_METHOD "pcap_sendpacket()"
 #endif
-
+#ifdef HAVE_LIBXDP
+#include <sys/mman.h>
+static sendpacket_t *sendpacket_open_xdp(const char *, char *) _U_;
+static struct tcpr_ether_addr *sendpacket_get_hwaddr_libxdp(sendpacket_t *);
```

Comment:
basically I copied sendpacket_get_hwaddr_pf and removed a couple of lines which I considered unnecessary

Reply:
nit: it became to common to refer for AF_XDP sockets as **xsk** (**X**DP **s**oc**k**et)
Its okay for now, but might consider use `sendpacket_open_xsk`

Reply:
agree